### PR TITLE
Change link to docs in product

### PR DESCRIPTION
**Pull request by olexii4**
### What does this PR do?
Fix link to factories sectiondocs in product:
/docs/integration-guide/workspace-automation/index.html -> /docs/factory/getting-started/index.htm

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/4502

#### Changelog
Change link to docs in product 

#### Release Notes
N/A

#### Docs PR
in progress

### CODE REVIEW

File: dashboard/src/components/widget/footer/che-footer.directive.ts [hunk 1]
```diff
@@ -87,7 +87,7 @@ export class CheFooter {
     }
 
     // docs
-    template += '<a class=\"che-footer-button-blue che-footer-button\" href=\"/docs\" target=\"_blank\">Docs<a/>';
+    template += '<a class=\"che-footer-button-blue che-footer-button\" href=\"/docs/factory/getting-started/index.html\" target=\"_blank\">Docs<a/>';
```
Comment (ashumilova) [inline dashboard/src/components/widget/footer/che-footer.directive.ts]: @olexii4 is it okay from your point of view, that link to all docs now points to factories section?

https://github.com/eclipse/che/blob/master/dashboard/src/app/factories/factory-details/information-tab/factory-information/factory-information.html#L179
https://github.com/eclipse/che/blob/master/dashboard/src/app/factories/list-factories/list-factories.html#L21

Comment (olexii4) [reply dashboard/src/components/widget/footer/che-footer.directive.ts]: No. I guessed  it is like general directory. It is clear now.

Comment (olexii4) [reply dashboard/src/components/widget/footer/che-footer.directive.ts]: Done

Comment (olexii4) [reply dashboard/src/components/widget/footer/che-footer.directive.ts]: It was 'Update links to docs in product'. That is why I changed general link in first time.

**Review by ashumilova — APPROVED** [review]

**Review by akurinnoy — APPROVED** [review]

**Pull request merged**